cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -261,31 +278,30 @@ func Test_TreePrefix(t *testing.T) {
 				return obectjTree
 			}(),
 			expectPrefix: []string{
-				"Object/root",
-				"├─Object/child1",
-				"│ │           ├─C1.1", // first condition child gets pipes, children pipe and ├─
-				"│ │           └─C1.2", // last condition child gets pipes, children pipe and └─
-				"│ └─Object/child1.1",
+				`"Object/root"
+				"├─Object/child1"
+				"│ │           ├─C1.1"
+				"│ │           └─C1.2"
+				"│ └─Object/child1.1"
 				"└─Object/child2",
-				"  │           ├─C2.1", // first condition child gets spaces, children pipe and ├─
-				"  │           └─C2.2", // last condition child gets spaces, children pipe and └─
-				"  └─Object/child2.1",
+				"  │           ├─C2.1"
+				"  │           └─C2.2"
+				"  └─Object/child2.1"`,
 			},
 		},
 	}
+
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			g := NewWithT(t)
+			var buf bytes.Buffer
 
 			// Creates the output table
-			tbl := uitable.New()
+			tbl := tablewriter.NewWriter(&buf)
```

Comment:
The current tests are not passing. Looks like you are missing a call to the `Render()` function.